### PR TITLE
fix latency issue and update error message

### PR DESCRIPTION
**Pull request by jaydeepsingh25**
The PR provides a fix for https://github.com/Shared-Reality-Lab/IMAGE-browser/issues/247 and  https://github.com/Shared-Reality-Lab/IMAGE-server/issues/468

### CODE REVIEW

File: src/errors/no_renderings.html [hunk 1]
```diff
@@ -8,6 +8,15 @@
             IMAGE is unable to render this graphic. If you would like to let us know about this, please send a link to this graphic and a short description of the problem via email: <a href = "mailto: image@cim.mcgill.ca"> image@cim.mcgill.ca</a> or report bugs using <a id="feedback-a" href="../feedback/feedback.html" target="_blank">our feedback form</a>.
```
Comment (jeffbl) [inline src/errors/no_renderings.html:8]: This paragraph should also be updated to the text in https://github.com/Shared-Reality-Lab/IMAGE-server/issues/468#issuecomment-1231930808

**Review by JRegimbal — APPROVED** [review]
Tested on Chromium. I don't see the long polling now, so this should be fine. To check, did you figure out why the array was initialized to be full of junk previously? It should be fine to merge as is in any case.

File: src/errors/no_renderings.html [hunk 2]
```diff
@@ -5,9 +5,19 @@
...
         </p>
+        <p>
+            Graphic file : (for example, photographs including .jpg, .png, or .gif) Scenes containing only unusual or oddly framed objects, montages of separate pictures, highly stylized or hand-drawn graphics, and diagrams are not yet well supported by the machine learning tools, and may not produce any usable renderings.
+        </p>
+        <p>
+            Charts: We only support a subset of charts formatted with the <a href="https://www.highcharts.com/">HighCharts</a> format. Currently, this means they can only be pie charts, or single-trend area or single-line charts. Further formats are under development.
+        </p>
+        <p>
+            Maps: Only maps where we can determine the latitude and longitude of the center of the map can generate renderings. If that location has no points of interest or streets nearby, IMAGE cannot generate any usable renderings. Formats such as Google Maps and Open Street Maps usually work compared to other map formats given these requirements.
```
Comment (jeffbl) [inline src/errors/no_renderings.html:19]:
```suggestion
            Maps: Only maps where we can determine the latitude and longitude of the center of the map can generate renderings. If that location has no points of interest or streets nearby, IMAGE cannot generate any usable renderings.
            Today, only Google Maps embedded in a page are supported, although other formats, such as Open Street Maps, may be supported in the future.
```

Comment (jeffbl) [reply src/errors/no_renderings.html:19]: Talked with @Cybernide and the above text is more reflective of current reality.

File: src/errors/no_renderings.html [hunk 3]
```diff
@@ -5,9 +5,19 @@
     </head>
     <body>
         <p>
-            IMAGE is unable to render this graphic. If you would like to let us know about this, please send a link to this graphic and a short description of the problem via email: <a href = "mailto: image@cim.mcgill.ca"> image@cim.mcgill.ca</a> or report bugs using <a id="feedback-a" href="../feedback/feedback.html" target="_blank">our feedback form</a>.
-            Please ensure that the graphic you are sending contains no personal information.
+            IMAGE is unable to render this graphic. If you would like to let us know about this, please use <a id="feedback-a" href="../feedback/feedback.html" target="_blank">our feedback form</a>. 
+            If you are unable to use the form, you can also send a link to this graphic and a short description of the problem via email: <a href = "mailto: image@cim.mcgill.ca"> image@cim.mcgill.ca</a>.
+            In either case, please ensure that the graphic you are sending contains no personal information.There are some common issues that prevent IMAGE from being able to generate a rendering.
```
Comment (jeffbl) [inline src/errors/no_renderings.html:10]:
```suggestion
            In either case, please ensure that the graphic you are sending contains no personal information.
            </p>
            <p>
            There are some common issues that prevent IMAGE from being able to generate a rendering, including:
```

**Pull request merged**